configs/en-lt-spring-2024.yml

```diff
@@ -125,10 +126,9 @@ datasets:
   - opus_ELRC-2740-vaccination/v1 #                             546 sentences
   - opus_ELRC-vaccination/v1 #                                  546 sentences
   - opus_ELRC_2923/v1 #                                         384 sentences
-  - mtdata_ELRC-lithuanian_legislation_seimas_lithuania-1-eng-lit
-  - mtdata_ELRC-eu_publications_medical_v2-1-eng-lit
-  - mtdata_ELRC-wikipedia_health-1-eng-lit
-  - mtdata_ELRC-nteu_tierb-1-eng-lit
+  - mtdata_ELRC-lithuanian_legislation_seimas_lithuania-1-eng-lit # ~611,715 sentences (69.1 MB)
```

Comment:
This one failed in all retries so I had to remove it. Maybe the  resource was temporarily unavailable